python/ray/serve/_private/controller.py

```diff
@@ -969,6 +988,10 @@ def delete_apps(self, names: Iterable[str]):
         for name in names:
             self.application_state_manager.delete_application(name)
 
+    def delete_apps_xlang(self, apps_bytes: bytes):
```

Comment:
unify this too?

Reply:
hi @liuyang-my , looks like list is supported. https://docs.ray.io/en/latest/ray-core/cross-language.html#cross-language-data-serialization.

Reply:
Deleted. Reuse `delete_apps`.